tools/e2etesting/DeployEdge.ps1

```diff
@@ -16,8 +18,15 @@ if (!$ResourceGroupName) {
     Write-Error "ResourceGroupName not set."
 }
 
-$edgeVmUsername = "sandboxuser"
-$edgeVmPassword = -join ((65..90) + (97..122) + (33..38) + (40..47) + (48..57)| Get-Random -Count 20 | % {[char]$_})
+if (!$KeysPath) {
```

Comment:
Why not use a temp directory (see DeployTestEventProcessor.ps1 lines 158/159)?

Reply:
WorkingDirectory is cleaned by build agent, this ensure the clean of generated keys.

Reply:
changed to use $(Agent.TempDirectory) , which is by definition cleaned after each build